WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -265,7 +265,12 @@ public void onClick(View v) {
         mBtnSpamComment.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                moderateComment(CommentStatus.SPAM);
+                if (mBtnSpamComment.getText().equals(
```

Comment:
What do you think of change this if-condition by checking the status of the post?
In the same way the label of the button is updated in the code:
```
            if (mComment.getStatusEnum() == CommentStatus.SPAM) {
                moderateComment(CommentStatus.APPROVED);
            } else {
               moderateComment(CommentStatus.SPAM);
            }
```


Note that we're moving the comment from the SPAM status to the APPROVED. Is it the same in the web version of WordPress?